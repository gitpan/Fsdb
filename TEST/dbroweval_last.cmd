prog='dbroweval'
args='-f TEST/dbroweval_last.code'
cmp='diff -cb '
