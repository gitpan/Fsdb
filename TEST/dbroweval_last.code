if (defined(_last_clock)) { my $ad = _clock - _last_clock; _absdiff = sprintf("%0.6f", $ad) }
