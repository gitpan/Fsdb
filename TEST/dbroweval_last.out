#fsdb event clock absdiff
_null_getpage+128	815812813.281756	-
_null_getpage+128	815812813.328709	0.046953
_null_getpage+128	815812813.353830	0.025121
_null_getpage+128	815812813.357169	0.003339
_null_getpage+128	815812813.375844	0.018675
_null_getpage+128	815812813.378358	0.002514
#  | /home/johnh/BIN/DB/dbrow 
#  | /home/johnh/BIN/DB/dbcol event clock
#   | dbcol -v pctdiff
#   | dbroweval   { if (defined(_last_clock)) { my $ad = _clock - _last_clock; _absdiff = sprintf("%0.6f", $ad) } ; }
